pmg/views.py

```diff
@@ -210,6 +210,14 @@ def committee_detail(committee_id):
     """
     logger.debug("committee detail page called")
     committee = load_from_api('committee', committee_id)
+    YEAR_CUTOFF = 2012
+    now = datetime.now()
+    meetings_by_year = {}
+
+    def format_date(m_date):
```

Comment:
This seems weird to me. What if we just grabbed the first 4 chars from the date string and used that as the year?